### PR TITLE
prototype: add smart account page and signature verification

### PR DESCRIPTION
**Pull request by PaddyMc**
## What is the purpose of the change

Prototype for spendlimits

### CODE REVIEW

**Review by MaxMillington — COMMENTED** [review]
TLDR 🤣

File: packages/stores/src/account/base.ts [hunk 1]
```diff
@@ -682,95 +693,185 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
...
-      value: signedTxBody,
-    };
-
-    const signedTxBodyBytes = wallet?.signingStargateOptions?.registry?.encode(
-      signedTxBodyEncodeObject
+    const sessionKey = localStorage.getItem("SessionKey");
+    if (sessionKey == null) {
+      throw new Error("session key not found or broken");
+    }
+    const privateKey = new PrivKeySecp256k1(fromBase64(sessionKey));
```
Comment (JoseRFelix) [inline packages/stores/src/account/base.ts:744]: Will this be able to support `PrivKeySecp256r1`?

**Review by sunnya97 — CHANGES_REQUESTED** [review]
asdf

**Comment by sunnya97** [comment]
- [ ] dsfdf

File: packages/proto-codecs/scripts/codegen.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ const includedPackages: AvailablePackages[] = [
   "osmosis.concentratedliquidity.v1beta1",
```
Comment (sunnya97) [inline packages/proto-codecs/scripts/codegen.ts:27]: This is broken

File: packages/stores/src/account/base.ts (hunk 1 above)
Comment (PaddyMc) [reply packages/stores/src/account/base.ts:744]: It should if we edit the code to generate an R1 signature, not sure if it's needed atm

**Review by JoseRFelix — APPROVED** [review]

**Pull request merged**